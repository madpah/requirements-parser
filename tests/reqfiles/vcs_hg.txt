-e hg+http://hg.myproject.org/MyProject#egg=MyProject
-e hg+https://hg.myproject.org/MyProject#egg=MyProject
-e hg+ssh://hg.myproject.org/MyProject#egg=MyProject
-e hg+http://hg.myproject.org/MyProject@da39a3ee5e6b#egg=MyProject
-e hg+http://hg.myproject.org/MyProject@2019#egg=MyProject
-e hg+http://hg.myproject.org/MyProject@v1.0#egg=MyProject
-e hg+http://hg.myproject.org/MyProject@special_feature#egg=MyProject

hg+http://hg.myproject.org/MyProject#egg=MyProject
hg+https://hg.myproject.org/MyProject#egg=MyProject
hg+ssh://hg.myproject.org/MyProject#egg=MyProject
hg+http://hg.myproject.org/MyProject@da39a3ee5e6b#egg=MyProject
hg+http://hg.myproject.org/MyProject@2019#egg=MyProject
hg+http://hg.myproject.org/MyProject@v1.0#egg=MyProject
hg+http://hg.myproject.org/MyProject@special_feature#egg=MyProject
