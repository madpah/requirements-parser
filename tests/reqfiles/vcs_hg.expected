[
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": true, 
  "uri": "hg+http://hg.myproject.org/MyProject", 
  "extras": [], 
  "vcs": "hg", 
  "path": null, 
  "line": "-e hg+http://hg.myproject.org/MyProject#egg=MyProject", 
  "specs": [], 
  "revision": null
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": true, 
  "uri": "hg+https://hg.myproject.org/MyProject", 
  "extras": [], 
  "vcs": "hg", 
  "path": null, 
  "line": "-e hg+https://hg.myproject.org/MyProject#egg=MyProject", 
  "specs": [], 
  "revision": null
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": true, 
  "uri": "hg+ssh://hg.myproject.org/MyProject", 
  "extras": [], 
  "vcs": "hg", 
  "path": null, 
  "line": "-e hg+ssh://hg.myproject.org/MyProject#egg=MyProject", 
  "specs": [], 
  "revision": null
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": true, 
  "uri": "hg+http://hg.myproject.org/MyProject", 
  "extras": [], 
  "vcs": "hg", 
  "path": null, 
  "line": "-e hg+http://hg.myproject.org/MyProject@da39a3ee5e6b#egg=MyProject", 
  "specs": [], 
  "revision": "da39a3ee5e6b"
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": true, 
  "uri": "hg+http://hg.myproject.org/MyProject", 
  "extras": [], 
  "vcs": "hg", 
  "path": null, 
  "line": "-e hg+http://hg.myproject.org/MyProject@2019#egg=MyProject", 
  "specs": [], 
  "revision": "2019"
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": true, 
  "uri": "hg+http://hg.myproject.org/MyProject", 
  "extras": [], 
  "vcs": "hg", 
  "path": null, 
  "line": "-e hg+http://hg.myproject.org/MyProject@v1.0#egg=MyProject", 
  "specs": [], 
  "revision": "v1.0"
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": true, 
  "uri": "hg+http://hg.myproject.org/MyProject", 
  "extras": [], 
  "vcs": "hg", 
  "path": null, 
  "line": "-e hg+http://hg.myproject.org/MyProject@special_feature#egg=MyProject", 
  "specs": [], 
  "revision": "special_feature"
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": false, 
  "uri": "hg+http://hg.myproject.org/MyProject", 
  "extras": [], 
  "vcs": "hg", 
  "path": null, 
  "line": "hg+http://hg.myproject.org/MyProject#egg=MyProject", 
  "specs": [], 
  "revision": null
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": false, 
  "uri": "hg+https://hg.myproject.org/MyProject", 
  "extras": [], 
  "vcs": "hg", 
  "path": null, 
  "line": "hg+https://hg.myproject.org/MyProject#egg=MyProject", 
  "specs": [], 
  "revision": null
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": false, 
  "uri": "hg+ssh://hg.myproject.org/MyProject", 
  "extras": [], 
  "vcs": "hg", 
  "path": null, 
  "line": "hg+ssh://hg.myproject.org/MyProject#egg=MyProject", 
  "specs": [], 
  "revision": null
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": false, 
  "uri": "hg+http://hg.myproject.org/MyProject", 
  "extras": [], 
  "vcs": "hg", 
  "path": null, 
  "line": "hg+http://hg.myproject.org/MyProject@da39a3ee5e6b#egg=MyProject", 
  "specs": [], 
  "revision": "da39a3ee5e6b"
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": false, 
  "uri": "hg+http://hg.myproject.org/MyProject", 
  "extras": [], 
  "vcs": "hg", 
  "path": null, 
  "line": "hg+http://hg.myproject.org/MyProject@2019#egg=MyProject", 
  "specs": [], 
  "revision": "2019"
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": false, 
  "uri": "hg+http://hg.myproject.org/MyProject", 
  "extras": [], 
  "vcs": "hg", 
  "path": null, 
  "line": "hg+http://hg.myproject.org/MyProject@v1.0#egg=MyProject", 
  "specs": [], 
  "revision": "v1.0"
 }, 
 {
  "specifier": false, 
  "local_file": false, 
  "name": "MyProject", 
  "editable": false, 
  "uri": "hg+http://hg.myproject.org/MyProject", 
  "extras": [], 
  "vcs": "hg", 
  "path": null, 
  "line": "hg+http://hg.myproject.org/MyProject@special_feature#egg=MyProject", 
  "specs": [], 
  "revision": "special_feature"
 }
]